packages/shopping-cart/src/types.ts

```diff
@@ -584,6 +584,8 @@ export interface ResponseCartProductVariant {
 	product_slug: string;
 	currency: string;
 	price_integer: number;
+	price_before_discounts_integer: number;
+	introductory_offer_terms: IntroductoryOfferTerms;
```

Comment:
Minor: this isn't a full implementation of `IntroductoryOfferTerms`, and it can be empty. I think this should do it:

```
	introductory_offer_terms: Record< string, never > | Pick< IntroductoryOfferTerms, 'interval_unit' | 'interval_count' >;
```

Reply:
Good catch.  Added that